src/components/ExplorePage/index.js

```diff
@@ -81,19 +105,24 @@ function ExplorePage({
       setGeography(g);
     }
   }, [data]);
+  const isLoading = shouldFetch && !(data || error);
   return (
     <div className={classes.root}>
-      <Hidden smDown>
-        <Map
-          center={[0.3051933453207569, 37.908818734483155]}
-          zoom={6.25}
-          geometries={geometries}
-          geography={geography}
-          setShouldFetch={setShouldFetch}
-          setGeoCode={setGeoCode}
-          {...props}
-        />
-      </Hidden>
+      <Map
+        center={[0.3051933453207569, 37.908818734483155]}
+        zoom={6.25}
+        geometries={geometries}
+        geography={geography}
+        setShouldFetch={setShouldFetch}
+        setGeoCode={setGeoCode}
+        {...props}
+        className={classes.map}
+      />
+      <Location
+        {...locationArgs}
```

Comment:
Bonus: We could use the geography state variable
![Screenshot from 2021-09-20 16-10-50](https://user-images.githubusercontent.com/7962097/134007948-2fde8df7-e617-4926-b352-425b9f219d17.png)
a

Reply:
In a sub-county/district @KhadijaMahanga , will parents be [_county_, **country**] or [**country**, _county_] order?